index.html

```diff
@@ -265,7 +268,7 @@ <h4 class="title">Favorites</h4>
       </div>
       <div class="main-content container-fluid">
         <div class="row">
-          <div class="col-md-12">
+          <div class="col-md-12 tool-bar-holder" id="tool-bar-holder">
```

Comment:
@EatPrayCode Why are we using both id and class here?

Reply:
There is some necessary to get by id, incases like hiding and showing the toolbar.
yes we can get by class, but then we need to do getByClass[0] which is little bit cumbersome.
Can we do this change once complete toolbox hiding functionality is done.